PHPCI/Application.php

```diff
@@ -133,7 +133,13 @@ protected function setLayoutVariables(View &$layout)
     {
         /** @var \PHPCI\Store\ProjectStore $projectStore */
         $projectStore = b8\Store\Factory::getStore('Project');
-        $layout->projects = $projectStore->getAll();
+        $layout->projects = $projectStore->getWhere(
+          array('archived' => (int) isset($_GET['archived'])),
```

Comment:
This mean that if ```archived``` is ```false``` or ```0``` we always display also archived projects because is set, eg:

    http://phpci/controller/action?archived=false

have ``isset($_GET['archived']);``` as true and can be confusing, maybe we need to check for isset and it's value.